recipes/font-ttf-dejavu-sans-mono-feedstock/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = '2.37' %}
+
+package:
+  name: font-ttf-dejavu-sans-mono
+  version: {{ version }}
+
+source:
+  url: https://sourceforge.net/projects/dejavu/files/dejavu/{{ version }}/dejavu-sans-ttf-{{ version }}.zip
+  sha256: 5c6e497a2f36552cb5ffb112c413a6af39c0f3c47653662b90b4fa6499822fd7
+
+build:
+  number: 0
+  noarch: generic
+
+test:
+  requires:
+    # dummy to have at least one test env requirement
+    - bzip2
+  commands:
+    - test -f ${PREFIX}/fonts/DejaVuSans.ttf  # [unix]
+
+about:
+  home: https://dejavu-fonts.github.io/
+  summary: Monospace font for pretty code listings with many mathematical and other symbols
+  description: |
+    DejaVu fonts are a font family based on the Vera Fonts. Its purpose is to provide a
+    wider range of characters while maintaining the original look and feel through the
+    process of collaborative development Monospace font for pretty code listings.
+    Next to that, DejaVu contains a lot of mathematical and other symbols, arrows, braille
+    patterns, etc
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
This seems to be incorrect, as the docs seem to say they are in the public domain: https://dejavu-fonts.github.io/License.html

Reply:
hm... this was taken over like that from AnacondaRecipes and looked correct. Seems like Dejavu fonts are based on multiple previous fonts (all `BSD` licensed) plus the dejavu work on top, which is `public domain`. Now also changed the meta.yaml to `license: Public Domain, BSD-3-Clause`, `license_family: OTHER`. Hope that is more correct. The included LICENSE file is anyway stating this all correct.

Reply:
Hmm it isn't BSD actually though. It can't be if it is public domain. And you shouldn't have a license file, per se, except maybe a copy of the website

Reply:
Yes, it can and it is @scopatz, please read the link you pasted carefully and compare most of the text with a BSD license. Apart from the part that states:

> DejaVu changes are in public domain

The rest of this license is *very clearly* BSD.

Reply:
> And you shouldn't have a license file, per se ..

Except for where it states: "The above copyright and trademark notices and this permission notice shall be included in all copies of one or more of the Font Software typefaces. "

We should use the license file from the link.

Reply:
I don't think public domain works like that though. Specifically, public domain (in the US) is the absence of a copyright or other IP claims.  It doesn't make sense for something to BSD + public domain.  BSD still requires the original copyright to be listed. 

> We should use the license file from the link.

Yep agreed

Reply:
I disagree and [Wikipedia](https://en.wikipedia.org/wiki/Public-domain_software) seems to as well:

> Paradoxically, his copyleft approach relies on the enforceability of the copyright to be effective. Copyleft free software shares therefore many properties with public-domain software, but doesn't allow relicensing or sublicensing. Unlike real public-domain software or permissive-licensed software, Stallman's copyleft license tries to enforce the free shareability of software also for the future by not allowing license changes.

Reply:
And many projects routinely copy public domain code into them and relicense it (by that very act).

Reply:
So I think that Deja Vu fonts are BSD-3 licensed, basically (and this was my thinking and research at the time).

Reply:
I am a little confused. I am saying that this is BSD licensed, under the notion that public domain works can be licensed as BSD later.  @mingwandroid - can you please clarify?

Reply:
We, as packagers/distributors have the same rights to relicense things where that is allowed by the respective licenes (or, lack of them).

Reply:
Ok Sounds good! Seems like we agree :wink:

Reply:
So what do we do? Reverting the last commit (which makes it `license: Public Domain, BSD-3-Clause`) and sticking to BSD ... or keeping it as it is?

Reply:
Ok. Done.